src/renderer/store.js

```diff
@@ -116,14 +115,7 @@ const persistOptions = {
   whitelist: ['subscriptions', 'publish', 'wallet', 'content', 'fileInfo', 'app'],
   // Order is important. Needs to be compressed last or other transforms can't
   // read the data
-  transforms: [
-    subscriptionsFilter,
-    walletFilter,
-    contentFilter,
-    fileInfoFilter,
-    appFilter,
-    compressor,
-  ],
+  transforms: [walletFilter, contentFilter, fileInfoFilter, appFilter, compressor],
```

Comment:
Why the change in formatting?
*Mostly curious

Reply:
I removed the `subscriptionsFilter` so prettier decided it could all be on one line